package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Rentzila",
+  "type": "module",
+  "scripts": {
+    "wdio-all": "npx wdio run ./wdio.conf.ts",
+    "wdio-authorization": "npx wdio run ./wdio.conf.ts --suite authorization"
```

Comment:
@Anastas11aZakharova Is that really required? I mean, the test case file is empty at all, must not be present in the PR (talking about the authorization.ts).

Reply:
Fixed in the latest commit